pandas/_libs/tslib.pyx

```diff
@@ -564,7 +562,7 @@ cpdef array_to_datetime(
                         elif is_raise:
                             raise ValueError(
                                 f"time data \"{val}\" at position {i} doesn't "
-                                f"match format \"{format}\""
+                                f"match ISO8601 standard"
```

Comment:
we only get here if `array_to_datetime` was called with `require_iso8601`, without a `format` (if it had a `format`, it would've gone to `strptime.pyx`)